modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -95,87 +82,553 @@ The following charts are available only if you are using Unified Agent and have
 * Outbound Operations
 * Outbound Protocol
 * Outbound Endpoints
-| *Flows* a|
-
-* Inbound Response Time by Flow: Average inbound response time
-* Outbound Response Time by Flow: Average outbound response time
-* Inbound Request Volume by Flow: Average number of inbound requests
-* Outbound Request Volume by Flow: Average number of outbound requests
-* Failures by Flow: Average number of failed inbound requests
-* Slow Inbound Requests by Flow: Average number of inbound requests whose response time is greater than 1000 ms
-* Slow Outbound Requests by Flow: Average number of outbound requests whose response time is greater than 1000 ms
-* Inbound Response Time by Flow: Average, minimum, and maximum inbound response time
-* Outbound Response Time by Flow: Average, minimum, and maximum outbound response time
-* Inbound Request Volume by Flow: Average number of inbound requests
-* Outbound Request Volume by Flow: Average number of outbound requests
-* Failures by Flow: Average number of inbound failed requests
-* Slow Inbound Requests by Flow: Average number of inbound requests whose response time is greater than 1000 ms
-* Slow Outbound Requests by Flow: Average number of outbound requests whose response time is greater than 1000 ms
+| <<flows-charts>> a|
+
+* Inbound Response Time by Flow
+* Outbound Response Time by Flow
+* Inbound Request by Flow
+* Outbound Request by Flow
+* Failures by Flow
+* Slow Inbound Requests by Flow
+* Slow Outbound Requests by Flow
+* Inbound Response Time by Flow
+* Outbound Response Time by Flow
+* Inbound Request Volume by Flow
+* Outbound Request Volume by Flow
```

Comment:
Looks like the same as above "Outbound Requests by Flow", just shown as a table rather than a chart.  If so, update the title to match.

@myuin-ms FYI:

Reply:
```suggestion
* Outbound Requests by Flow
```